packages/govern-server/src/core/config.ts

```diff
@@ -1,22 +1,22 @@
-export default {
-  networks: [
-    {
-      chainId: 1,
-      name: 'ethereum',
-      ensAddress: '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
-      subgraphId: 'aragon/aragon-govern-mainnet',
-    },
-    {
-      chainId: 4,
-      name: 'rinkeby',
-      ensAddress: '0x98df287b6c145399aaa709692c8d308357bc085d',
-      subgraphId: 'aragon/aragon-govern-mainnet',
-    },
-    {
-      chainId: 100,
-      name: 'xdai',
-      ensAddress: '0xaafca6b0c89521752e559650206d7c925fd0e530',
-      subgraphId: 'aragon/aragon-govern-mainnet',
-    },
-  ],
-}
+export const networks = [
+  {
+    chainId: 1,
+    name: 'ethereum',
+    ensAddress: '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
+    subgraphUrl:
+      'https://api.thegraph.com/subgraphs/name/aragon/aragon-govern-mainnet',
+  },
+  {
+    chainId: 4,
+    name: 'rinkeby',
+    ensAddress: '0x98df287b6c145399aaa709692c8d308357bc085d',
+    subgraphUrl:
+      'https://api.thegraph.com/subgraphs/name/aragon/aragon-govern-rinkeby',
+  },
+  // {
```

Comment:
Maybe we can keep a TODO to add xDai instead of keep the code commented?

Reply:
Done https://github.com/aragon/govern/pull/125/commits/b9b5871d014273494f9a3407e65970ee9c04c1b0